packages/tezos-lwt-result-stdlib/tezos-lwt-result-stdlib.8.1/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+maintainer: "contact@tezos.com"
+authors: [ "Tezos devteam" ]
+homepage: "https://www.tezos.com/"
+bug-reports: "https://gitlab.com/tezos/tezos/issues"
+dev-repo: "git+https://gitlab.com/tezos/tezos.git"
+license: "MIT"
+depends: [
+  "ocaml" { < "4.12" }
```

Comment:
Quick question: The release of OCaml 4.12 is getting close. Is there any chance of seeing this unlocked?

Reply:
`tezos-protocol-compiler` has strong constraint on the version of the OCaml compiler it uses and for now, development is still in 4.09!
We are already compatible with 4.10 and we'll work in the next month to move all the dev to OCaml 4.11. Maybe, once this done and we don't have anymore to be compatible with 4.09, we will be able to enable 4.12 compatibility here yes.